dev-util/plan9port/plan9port-20200330.ebuild

```diff
@@ -99,34 +112,24 @@ src_compile() {
 }
 
 src_install() {
-	dodir "${PLAN9}"
+	readme.gentoo_create_doc
 
-	# P9P's man does not handle compression
-	docompress -x "${PLAN9}/man"
+	# cleanup
+	rm -f LOCAL.config install.{log,sum}
```

Comment:
Don't use `-f`. You need to know whether these files exist or not.

Reply:
```
/var/db/repos/gentoo % rg -l -g'*.ebuild' 'rm -r?f' | wc -l
2215
```
I don't get why this is a problem.

Reply:
It makes users/maintainers miss errors with their ebuilds. ::gentoo is not always the best place to look at, most of the ebuilds are horrible and outdated. `-f` is only acceptable when something is not fully predictable.